### DISCUSSION TITLE
using a homepod mini as homekit hub

### DISCUSSION THREAD

**Discussion by marioveyna** [taligentx/dscKeybusInterface#238] [dscKeybusInterface Discussion]
hi, i want to implement for homekit but i have a homepod mini and want to use it instead homebridge, can do this? thanks

**Comment by taligentx** [comment]
Hi @marioveyna - you'll still need to use something like Homebridge to get the security system integrated into Homekit.  Then you can use the HomePod mini to control the security system as you would with Siri.

Side note, I've been looking at Arduino HomeKit libraries that can run entirely on esp8266/esp32 (without a separate computer running HomeBridge) and so far [HomeSpan](https://github.com/HomeSpan/HomeSpan) seems to be working well with dscKeybusInterface.  Downside is that it's currently esp32-only.
